dell/xps/15-7590/default.nix

```diff
@@ -0,0 +1,34 @@
+{ lib, ... }:
+
+{
+  imports = [
+    ../../../common/cpu/intel
+    ../../../common/pc/laptop
```

Comment:
Do they still provide non-ssd options for this model? Otherwise we could also import the ssd profile.